server/src/main/java/de/uftos/repositories/ucdl/parser/PredefinedConstraint.java

```diff
@@ -0,0 +1,124 @@
+package de.uftos.repositories.ucdl.parser;
+
+import de.uftos.dto.ResourceType;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.Getter;
+
+
+/**
+ * This enum models predefined constraints which are ready-to-use.
+ */
+@SuppressWarnings("checkstyle:LineLength")
+@Getter
+public enum PredefinedConstraint {
```

Comment:
I am not sure if an enum is the best type here. Also, what do you think about reading the default constraints from a file?

Reply:
I think an enum is easier to work with inside java as you don't have tons of "magic-Strings" everywhere.
I think a file would also work, but that would probably include a lot of implementation work.

Reply:
Ok, that works for me then :)